### ISSUE TITLE
Please teach me how to use it

### ISSUE THREAD

**Issue by soushu** [MessageKit/MessageInputBar#9]
How do I use this library? Is there a sample?

**Comment by nathantannar4** [comment]
Yes there is an Example! We are still working on the full docs. You could checkout the docs in https://github.com/nathantannar4/InputBarAccessoryView for now. This is a fork of that. There are some naming differences though

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
